lib/MusicBrainz/Server/Entity/CollectionType.pm

```diff
@@ -21,6 +21,15 @@ sub l_name {
     return lp($self->name, 'collection_type')
 }
 
+around TO_JSON => sub {
+    my ($orig, $self) = @_;
+
+    return {
+        %{ $self->$orig },
+        item_entity_type => $self->item_entity_type,
```

Comment:
(Seems it's already in the Flow type since https://github.com/metabrainz/musicbrainz-server/commit/c53b65015d71543e44d7dbd44e1d2078208f30bd but wasn't actually added to the output until now.)